packages/backend-core/src/db/couch/DatabaseImpl.ts

```diff
@@ -109,14 +109,35 @@ export class DatabaseImpl implements Database {
     }
   }
 
-  async get<T>(id?: string): Promise<T | any> {
+  async get<T extends Document>(id?: string): Promise<T> {
     const db = await this.checkSetup()
     if (!id) {
       throw new Error("Unable to get doc without a valid _id.")
     }
     return this.updateOutput(() => db.get(id))
   }
 
+  async getMultiple<T extends Document>(
+    ids: string[],
+    opts?: { failIfMissing?: boolean }
```

Comment:
nit: `failIfAnyMissing` would be a slightly clearer name, but I wouldn't block this PR on it.

Reply:
I also feel slightly stronger that the default should be `true`. It would surprise me if the majority of places this is used would be fine if they got an empty array back.

Reply:
funnily enough - the expectation of everywhere this is used today is to fetch what they can - but I do agree it would be better to be flipped, I'll set that and then add the option to all current calls.